### DISCUSSION TITLE
In language bar languages i used in my organizations don't appear

### DISCUSSION THREAD

**Discussion by fouad1233** [anuraghazra/github-readme-stats#1969] [Q&A] [answered]
**Describe the bug**
In language bar languages i used in my organizations don't appear, just language I used in my repos appear , and it's so important for me to see the languages I used in my organization because I have many teams.
So can you fix that? or do you have any other way to show languages I used in my organizations in my bar. 
Thank you.

**Comment by rickstaa** [comment, marked as answer]
@fouad1233 Thanks for your question. We currently do not include stats from repositories contained within organizations but only the ones a user owns (see for more information). This is a highly requested feature see https://github.com/anuraghazra/github-readme-stats/issues/1 which will likely be merged in the future. You can already use it on your own Vercel instance. See https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 for more information.

**Comment by fouad1233** [comment]
Thank you for your answer. I hope you include this feature in the future. Good luck! I will try to use it in my Vercel instance.
